protobuf-lite/build.gradle

```diff
@@ -18,6 +18,11 @@ dependencies {
             libraries.guava
 
     testProtobuf libraries.protobuf
+    // TODO(zpencer): fix proto plugin wrt testProtobuf and intellij
+    // Using testProtobuf to add the protobuf libriaries did not add the built-in
+    // types like com.google.protobuf.Empty to intellij's class path.
+    // Adding testCompile is a workaround.
+    testCompile libraries.protobuf
```

Comment:
This is dangerous, as protobuf and protobuf_lite contain some of the same classes, but with different versions.

Reply:
Valid point. I just discovered that these classes exist in `build/generated` so we should probably just be including them.